packages/flame_spine/assets/spineboy/spineboy.atlas

```diff
@@ -0,0 +1,307 @@
+
```

Comment:
Why are these assets needed at the root level of the package?

Reply:
in order to have tests, the skeleton animations needed to be in the package/assets directory since the underlying lib is not allowing to load from somewhere else.

I tried with mockito, change animation test not happening with mokito, but others are running ok.

Reply:
previous commit, tests with mock
[30e26dd](https://github.com/flame-engine/flame/pull/2109/commits/30e26dd5f018bf78c00a7815f319d353ffca4a5d)